dev/dev/devDeployment.yaml

```diff
@@ -49,6 +49,7 @@ datarepo-api:
     SENTRY_ENABLED: true
     SENTRY_ENVIRONMENT: development
     DUOS_SYNCUSERS_SCHEDULE: "@hourly"
+    DATAREPO_COMPACTIDPREFIXALLOWLIST_0_: foo.0
```

Comment:
TIL that converting property paths to environment variables requires that brackets be transformed to underscores :)

Reply:
Haha, yeah, that took a minute to figure out